lib/file-store.js

```diff
@@ -33,10 +33,41 @@ FileStore.prototype.load = function (fn) {
     if (!json || json === '') {
       return fn(null, {})
     }
+
+    let store
     try {
-      fn(null, JSON.parse(json))
+      store = JSON.parse(json)
     } catch (err) {
-      fn(err)
+      return fn(err)
     }
+
+    // Check if old format and convert if needed
+    if (!store.hasOwnProperty('lastRun') && store.hasOwnProperty('pos')) {
+      if (store.pos === 0) {
+        store.lastRun = null
+      } else {
+        if (store.pos > store.migrations.length) {
```

Comment:
Hmm, this would be indicative of a bigger problem right?  Maybe better to error in this case?